docs(ospo): community health rollout v2 — README, agents.md, health files (#397)
owncloud/notifications: README.md

# Notifications

<!-- OSPO-managed README | Generated: 2026-04-16 | v2 -->

[![License](https://img.shields.io/badge/License-AGPL--3.0-blue.svg)](COPYING) [![ownCloud OSPO](https://img.shields.io/badge/OSPO-ownCloud-blue)](https://kiteworks.com/opensource) [![Docker Hub](https://img.shields.io/docker/pulls/owncloud)](https://hub.docker.com/r/owncloud/server)

> The notifications backend and UI for ownCloud Server. It provides the notification bell/panel in the web interface and an OCS REST API that other ownCloud apps (such as announcementcenter and federatedfilesharing) use to create, display, and manage user notifications.

Reviewer: ```suggestion
The notifications backend and UI for ownCloud Classic. It provides the notification bell/panel in the web interface and an OCS REST API that other ownCloud apps (such as announcementcenter and federatedfilesharing) use to create, display, and manage user notifications.
```

## Getting Started

Follow the steps below to install and enable the Notifications app.

### Prerequisites

- ownCloud Server 10.x
- PHP 7.4+

### Installation

> This app is typically bundled with ownCloud Server. To install manually:

Reviewer: ```suggestion
This app is typically bundled with ownCloud Classic. To install manually:
```

```bash
git clone https://github.com/owncloud/notifications.git /var/www/owncloud/apps/notifications
occ app:enable notifications
```

### Build

```bash
make dist
```

### Run Tests

```bash
make test-php-unit
make test-php-style
```

## Documentation

- [Notification Workflow (for app developers)](https://github.com/owncloud/notifications/blob/master/docs/notification-workflow.md)
- [OCS Endpoint v1 (for client developers)](https://github.com/owncloud/notifications/blob/master/docs/ocs-endpoint-v1.md)

## Part of ownCloud Server (Classic)

> This app provides the core notification infrastructure for [ownCloud Server 10](https://github.com/owncloud/core). Other apps register notification providers through this app's API.

Reviewer: ```suggestion
This app provides the core notification infrastructure for [ownCloud Classic](https://github.com/owncloud/core). Other apps register notification providers through this app's API.
```

- **Dependabot**: Automated dependency updates are managed via Dependabot. Review and merge dependency PRs promptly.
- **Signed Commits**: All commits **must** be PGP/GPG signed. See [GitHub's signing guide](https://docs.github.com/en/authentication/managing-commit-signature-verification).
- **DCO Sign-off**: Every commit must carry a `Signed-off-by` line:
  ```
  git commit -s -S -m "your commit message"
  ```
- **GitHub Actions Policy**: Workflows may only use actions that are (a) owned by `owncloud`, (b) created by GitHub (`actions/*`), or (c) verified in the GitHub Marketplace.

## Translations

Help translate this project on Transifex:
**<https://explore.transifex.com/owncloud-org/owncloud/>**

Please submit translations via Transifex -- do not open pull requests for translation changes.

## Security

**Do not open a public GitHub issue for security vulnerabilities.**

Report vulnerabilities at **<https://security.owncloud.com>** -- see [SECURITY.md](SECURITY.md).

Bug bounty: [YesWeHack ownCloud Program](https://yeswehack.com/programs/owncloud-bug-bounty-program)

> 

Reviewer: ```suggestion
```

No longer exists.
